<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DragonDropTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Drop area</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0" columnstretch="1,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonDropClip">
        <property name="text">
         <string>Copy to clipboard</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDropPossible">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonDropOpen">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonDropSave">
        <property name="text">
         <string>Save...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DropArea" name="labelDrop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>&lt;drop content&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDropSuggested">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QTreeView" name="listDrop">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QPushButton" name="buttonDropToDrag">
        <property name="text">
         <string>Add to drag sources...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Drag sources</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0" columnstretch="1,0">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="DragSource" name="labelDrag">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>&lt;drag content&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonDragEdit">
        <property name="text">
         <string>Edit...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonDragLoad">
        <property name="text">
         <string>Load config...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelDragDefault">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonDragSave">
        <property name="text">
         <string>Save config...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDragSupported">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QListView" name="listDrag"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropArea</class>
   <extends>QLabel</extends>
   <header>droparea.h</header>
  </customwidget>
  <customwidget>
   <class>DragSource</class>
   <extends>QLabel</extends>
   <header>dragsource.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
